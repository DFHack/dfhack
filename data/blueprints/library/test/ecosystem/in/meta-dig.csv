#ignore
tests recursive meta composition, z-level manipulation, and repeat up/down

#dig label(center_tile) hidden()

,d,
#dig label(even_up) hidden()

u,,u
#dig label(even_down) hidden()
,j

,j
#meta label(even) hidden()
/even_up
/even_down
#dig label(odd_up) hidden()
,u

,u
#dig label(odd_down) hidden()

j,,j
#meta label(odd) hidden()
/odd_up
/odd_down
#meta label(twoz) hidden()
/even
#>
/odd
#meta label(fourz) hidden()
/twoz
#>2
/twoz
#>
/center_tile repeat(up 4)
#meta label(dig)
/fourz repeat(>3)
