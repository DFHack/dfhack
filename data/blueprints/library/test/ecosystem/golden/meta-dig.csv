#dig label(dig)
,j
u,d,u
,j
#>
,u
j,d,j
,u
#>
,j
u,d,u
,j
#>
,u
j,d,j
,u
#>
,j
u,d,u
,j
#>
,u
j,d,j
,u
#>
,j
u,d,u
,j
#>
,u
j,d,j
,u
#>
,j
u,d,u
,j
#>
,u
j,d,j
,u
#>
,j
u,d,u
,j
#>
,u
j,d,j
,u
